ddtrace/contrib/aiopg/patch.py

```diff
@@ -0,0 +1,152 @@
+# 3p
+import asyncio
+import aiopg.connection
+from aiopg.utils import _ContextManager
+import functools
+import wrapt
+import psycopg2.extensions
+
+from ddtrace.contrib import dbapi
+from ddtrace.contrib.psycopg.patch import _patch_extensions, \
+    patch_conn as psycppg_patch_conn
+from ddtrace.ext import sql
+from ddtrace import Pin
+
+
+# Original connect method, we don't want the _ContextManager
+_connect = aiopg.connection._connect
+
+
+class AIOTracedCursor(wrapt.ObjectProxy):
+    """ TracedCursor wraps a psql cursor and traces it's queries. """
+
+    _datadog_pin = None
+    _datadog_name = None
+
+    def __init__(self, cursor, pin):
+        super(AIOTracedCursor, self).__init__(cursor)
+        self._datadog_pin = pin
+        name = pin.app or 'sql'
+        self._datadog_name = '%s.query' % name
+
+    @asyncio.coroutine
+    def _trace_method(self, method, resource, extra_tags, *args, **kwargs):
+        pin = self._datadog_pin
```

Comment:
I think it's better to stick to our API so maybe:
```python
pin = Pin.get_from(self)
if not pin or not pin.enabled():
  # ...
```
In that way you can remove the class `_datadog_pin` attribute. Sounds good?